src/web/mjs/connectors/NitroScans.mjs

```diff
@@ -7,6 +7,10 @@ export default class NitroScans extends WordPressMadara {
         super.id = 'nitroscans';
         super.label = 'Nitro Manga';
         this.tags = [ 'webtoon', 'english' ];
-        this.url = 'https://nitromanga.com';
+        this.url = 'https://nitroscans.net';
+    }
+
+    async _initializeConnector() {
+        // do nothing on purpose
```

Comment:
What is the purpose?

Reply:
We do that in case website javascript "new" prevent our old electron to load mainpage. 
Its a shame ,since the rest of the code works